admin/client/App/sagas/index.js

```diff
@@ -8,8 +8,9 @@ import { loadItems } from '../screens/List/actions';
 /**
  * Load the items
```

Comment:
Looking at this, I'd probably argue dividing it, leaving the old `loadTheItems` for `actions. INITIAL_LIST_LOAD`, and a second one for filter actions, which does what the new function does. Don't have a strong name for it though.